src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+            String to = descriptor.split("/to ")[1];
+            String from = descriptor.split(" /")[1];
+            descriptor = descriptor.split("/")[0];
+            t = new Event(descriptor, from, to);
+        } else {
+            throw new IndexOutOfBoundsException();
+        }
+        inputList[numTasks+1] = t; //1-index
+        numTasks++;
+        printAddTask(t);
+    }
+
+    public static void printList(Task[] input) {
+        System.out.println(line + "\nHere are the tasks in your list: ");
+        for (int i = 1; i < input.length; i++) {
+            if (input[i] == null) {
+                break;
+            }
+            System.out.println(i + ". " + input[i]);
+        }
+        System.out.println(line);
+    }
+
+    public static void getCommand() {
+        Scanner in = new Scanner(System.in);
+        String userInput = in.nextLine();
+
+        while (true) {
+            if (userInput.equals("bye")) {
+                break;
+            } else if (userInput.equals("list")) {
+                printList(inputList);
+            } else if (userInput.contains("mark")) {
+                try {
+                    validTask(userInput);
+                } catch (DukeException e) {
+                    System.out.println("OOPS... task does not exist");
+                }
+            } else {
+                try {
+                    addTask(userInput);
+                } catch (DukeException e) {
+                    System.out.println("OOPS... The description of a " + userInput + " cannot be empty.");
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("OOPS... I'm sorry, but I don't know what that means :^(");
+                }
+            }
+            userInput = in.nextLine();
+        }
+        in.close();
+    }
+
+    public static void greet() {
+        String line = "__________________________________________________________";
+        System.out.println(line);
+        System.out.println("Hello! i'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println(line);
+    }
+
+    public static void bye() {
+        System.out.println(line);
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println(line + '\n');
+        return;
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+//        String logo = " ____        _        \n"
+//                + "|  _ \\ _   _| | _____ \n"
+//                + "| | | | | | | |/ / _ \\\n"
+//                + "| |_| | |_| |   <  __/\n"
+//                + "|____/ \\__,_|_|\\_\\___|\n";
+//        System.out.println("Hello from\n" + logo);
```

Comment:
Should try not to leave behind comments that provide no required information. You can refer to GitHub if you need the comments back.